test/functional/autoupdate/aup-basic-tests.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+	create_test_environment "$TEST_NAME"
+
+	# Path to install os-install in
+	run sudo sh -c "sudo mkdir '$TEST_DIRNAME'/testroot"
```

Comment:
I don't think you need to create this dir , if you need to install a real bundle into a directory you can install it in "$TARGETDIR" this is supposed to be the target system created by the test environment. However I think this test may be problematic because it is actually installing real bundle so it will need internet connectivity, it is not self contained as the rest are. I understand you need the real bundle so you will have the services there, but that makes me think this test doesn't belong here, maybe belongs to the real content tests https://github.com/clearlinux/swupd-client/tree/master/test/real_content, what do you think @otaviobp

Reply:
Thats a great observation, Thanks for catching that!

Reply:
Looks like I cant even get to pass it on my travis without having systemd daemon installed..I getting this failure: https://travis-ci.org/clearlinux/swupd-client/jobs/596251676#L8185

Reply:
Done with mocking systemctl ! Thanks for the tip